streams/src/main/java/org/apache/kafka/streams/kstream/Window.java

```diff
@@ -18,36 +18,56 @@
 package org.apache.kafka.streams.kstream;
 
 /**
- * A single window instance, defined by its start and end timestamp.
+ * A single window instance, defined by its start and end timestamp. TODO ?
  */
 public abstract class Window {
 
     private long start;
     private long end;
 
+    /**
+     * Create a new window for the given start time (inclusive) and end time (exclusive).
+     *
+     * @param start  the start timestamp of the window (inclusive)
```

Comment:
We should also document the relevant time unit (here: milliseconds) through this PR.